test/es6/default-splitscope.js

```diff
@@ -315,7 +315,36 @@ var tests = [
                 return y; 
             } 
         } 
-        assert.areEqual(10, (new c4()).f({})(), "The method defined as the default destructured value of the parameter should capture the formal from the param scope"); 
+        assert.areEqual(10, (new c4()).f({})(), "The method defined as the default destructured value of the parameter should capture the formal from the param scope");
+        
+        function f3(a = 10, d, b = (function () { return a; }, class { method1() { return a; } }), c) { 
```

Comment:
could you have test case where class has a name and also class has a 'extends' ?